src/scenes/Game.ts

```diff
@@ -20,36 +27,129 @@ export default class Game extends Phaser.Scene {
     createCharacterAnims(this.anims)
 
     const map = this.make.tilemap({ key: 'tilemap' })
-    const ground1 = map.addTilesetImage('Room_Builder_Office', 'tiles1')
-    const ground2 = map.addTilesetImage('Room_Builder_Floors', 'tiles2')
-    const ground3 = map.addTilesetImage('Room_Builder_Walls', 'tiles3')
-    const obj1 = map.addTilesetImage('Generic', 'tiles4')
-    const obj2 = map.addTilesetImage('Modern_Office_Black_Shadow', 'tiles5')
-    const obj3 = map.addTilesetImage('Classroom_and_library', 'tiles6')
-
-    const ground = [ground1, ground2, ground3, obj1]
-    const office_obj = [obj1, obj2, obj3]
-
-    const groundLayer = map.createLayer('Ground', ground)
-    map.createLayer('Obj_layer1', office_obj)
-    map.createLayer('Obj_layer2', office_obj)
-    map.createLayer('Obj_layer3', office_obj)
+    const FloorAndGround = map.addTilesetImage('FloorAndGround', 'tiles_wall')
 
+    const groundLayer = map.createLayer('Ground', FloorAndGround)
     groundLayer.setCollisionByProperty({ collides: true })
 
     // debugDraw(groundLayer, this)
 
+    // import wall objects from Tiled map to Phaser
```

Comment:
This file and specifically this constructor seems to be the messiest part of the code base for now. Since things can change a lot in the early stage, we can certainly clean this up in the future. (optional: for some immediate easy cleanups, you can start by grouping logics into smaller helper functions, one guideline is that most functions should be doing one single thing, and should not be too long and hard to track.)

Reply:
we can do this refactor in the future